tron/mesos.py

```diff
@@ -0,0 +1,272 @@
+import json
+import logging
+from urllib.parse import urlparse
+
+import requests
+from task_processing.runners.subscription import Subscription
+from task_processing.task_processor import TaskProcessor
+from twisted.internet.defer import logError
+
+from tron.actioncommand import ActionCommand
+from tron.utils.dicts import get_deep
+from tron.utils.queue import PyDeferredQueue
+
+TASK_LOG_FORMAT = '%(asctime)s %(name)s %(levelname)s %(message)s'
+# TODO: put in configs
+MESOS_MASTER_PORT = 5050
+DOCKERCFG_LOCATION = "file:///root/.dockercfg"
+MESOS_SECRET = ''
+MESOS_ROLE = '*'
```

Comment:
I wonder if, in the future, we'll want to make this a per task thing. From the framework's point of you, it has to tell the master which roles it wants to subscribe to, but I think it can re-register with more roles (so you could re-register when an action with a previously-unseen role is added). Fine to pass on this for now and figure it out later, was just a thought.